### ISSUE TITLE
A way to reference the original mocha method

### ISSUE THREAD

**Issue by Robdel12** [thefrontside/bigtest#64]
Currently you can reference `window.it` or `global.it` to get the original mocha method, but we should probably have a nicer way for going about this.

I need this for running a11y tests. The a11y checker looks at the rendered dom and reports violations back. This can't be run in a convergence.

**Comment by cowboyd** [comment]
So is the deal that the a11y checker has side-effects?

**Comment by Robdel12** [comment]
I imagine that this will be a thing that comes up from the greater community too. 

I was able to get around those side-effects:

```js
// Wrap Axe in a promise for a nicer API
let runAxe = new Promise((resolve, reject) => {
  axe.run((err, results) => {
    if (err) reject(err);
    resolve(results);
  });
});

export function checkA11y() {
  describe('Axe accessibility check', () => {
    let err, results;

    beforeEach(() => {
      err = results = null;
      return runAxe.then(axeResults => {
        results = axeResults;
      }).catch(error => {
        err = error;
      });
    });

    it('passes axe accessibility check', () => {
      expect(err).to.not.exist;
      expect(results).to.not.be.empty;
      expect(results.violations.length).to.have.lengthOf(0, `${printA11yViolations(results)}`);
    });
  });
}
```
